HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # TileDB v2.20.1 Release Notes
 
+## Defects removed
+
+* Support reading V1 group details with explicit version in the name. [#4744](https://github.com/TileDB-Inc/TileDB/pull/4744)
```

Comment:
```suggestion
* Fix reading groups created with versions < 2.15 after being modified by versions >= 2.19 [#4744](https://github.com/TileDB-Inc/TileDB/pull/4744)
```

I think my original release notes entry communicates the fixed defect better to the user.

Reply:
It was already merged in dev like this :(